docs/data/toolpad/getting-started/overview.md

```diff
@@ -2,6 +2,8 @@
 
 <p class="description">Toolpad is a low-code UI builder for creating apps with MUI's suite of component libraries.</p>
 
+{{"component": "components/landing/Banner.js", "action": "tryDemo", "category": "ToolpadDocs", "description": "Check out Toolpad's interactive demo. No sign-up required!", "href": "https://demo.toolpad.io", "label": "Try Demo", "docs": true }}
```

Comment:
How does this work? :D Is it intended?

Reply:
This happens through https://github.com/mui/material-ui/blob/master/docs/packages/markdown/loader.js where we allow `{{ "component": .. }}` and `{ "demo": ...}}` to support components inside the docs